### ISSUE TITLE
Installation fails due to conflicting cryptography version

### ISSUE THREAD

**Issue by NeolithEra** [graysonhead/runcible#2]
@graysonhead Hi, users are unable to run **_runcible_** due to dependency conflict with _**cryptography**_ package. As shown in the following full dependency graph of **_runcible_**, it directly requires _**cryptography==2.4.2**_ , while **_paramiko_** requires _**cryptography>=2.5**_ .

According to pip’s “first found wins” installation strategy, _**cryptography==2.4.2**_ is the actually installed version. However, _**cryptography==2.4.2**_ does not satisfy _**cryptography>=2.5**_.

### Dependency tree------
```
----------------------------------------------------------------------------------------
Building runcible-0.0.5
----------------------------------------------------------------------------------------
runcible<version range:>
| +-colorama<version range:>=0.4.1>
| +-**_cryptography<version range:==2.4.2>_**
| +-mergedb<version range:>=0.0.11>
| | +-alabaster<version range:==0.7.12>
| | +-babel<version range:==2.6.0>
| | +-bleach<version range:==3.1.0>
| | +-certifi<version range:==2019.3.9>
| | +-chardet<version range:==3.0.4>
| | +-colorama<version range:==0.4.1>
| | +-coverage<version range:==4.5.3>
| | +-docutils<version range:==0.14>
| | +-idna<version range:==2.8>
| | +-imagesize<version range:==1.1.0>
| | +-jinja2<version range:==2.10.1>
| | +-markupsafe<version range:==1.1.1>
| | +-packaging<version range:==19.0>
| | +-pkginfo<version range:==1.5.0.1>
| | +-pprint<version range:==0.1>
| | +-pygments<version range:==2.4.0>
| | +-pyparsing<version range:==2.4.0>
| | +-pytz<version range:==2019.1>
| | +-pyyaml<version range:==5.1>
| | +-readme-renderer<version range:==24.0>
| | +-requests<version range:==2.22.0>
| | +-requests-toolbelt<version range:==0.9.1>
| | +-six<version range:==1.12.0>
| | +-snowballstemmer<version range:==1.2.1>
| | +-sphinx<version range:==2.0.1>
| | +-sphinx-rtd-theme<version range:==0.4.3>
| | +-sphinxcontrib-applehelp<version range:==1.0.1>
| | +-sphinxcontrib-devhelp<version range:==1.0.1>
| | +-sphinxcontrib-htmlhelp<version range:==1.0.2>
| | +-sphinxcontrib-jsmath<version range:==1.0.1>
| | +-sphinxcontrib-programoutput<version range:==0.14>
| | +-sphinxcontrib-qthelp<version range:==1.0.2>
| | +-sphinxcontrib-serializinghtml<version range:==1.1.3>
| | +-tqdm<version range:==4.32.1>
| | +-twine<version range:==1.13.0>
| | +-urllib3<version range:==1.25.3>
| | +-webencodings<version range:==0.5.1>
| +-paramiko<version range:>=2.4.2>
| | +-bcrypt<version range:>=3.1.1>
| | +-**_cryptography<version range:>=2.5>_**
| | +-pynacl<version range:>=1.0.1>
| +-pyyaml<version range:>=0.5.1>
-------

```

### Solution------
1. First solution: you can revise your direct dependency **_cryptography==2.4.2_** to **_cryptography>=2.4.2_**.
2. Second solution: revise your direct dependency **_paramiko<version range:>=2.4.2_** to **_paramiko>=2.4.2,<2.5_**. Then, it will transitively introduces **_cryptography>=1.5_**, which solves the conflicts.

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
![6](https://user-images.githubusercontent.com/52778917/61120697-26a4d400-a4d0-11e9-9f8c-a30911d4f253.png)

**Comment by graysonhead** [comment]
Thanks for raising this issue. This is fixed in 0.0.6.
